bottles/backend/managers/backup.py

```diff
@@ -191,7 +192,7 @@ def duplicate_bottle(config, name) -> Result:
         """Duplicates the bottle with the specified new name."""
         logging.info(f"Duplicating bottle: {config.get('Name')} to {name}")
 
-        path = name.replace(" ", "_")
+        path = pathvalidate.sanitize_filename(name, platform="universal")
```

Comment:
```suggestion
        path = pathvalidate.sanitize_filename(name, platform="Linux")
```

Reply:
I'm more inclined to use universal, so that if bottles are transferred to a NTFS filesystem for example, they keep their name

Reply:
Good point.